### ISSUE TITLE
Breadcrumbs overflow list keyboard navigation

### ISSUE THREAD

**Issue by evansjohnson** [palantir/blueprint#6869]
#### Steps to reproduce
https://blueprintjs.com/docs/#core/components/breadcrumbs

Attempt to tab to open overflow list

#### Actual behavior
Cannot tab to popover menu target

#### Expected behavior
Should be able to access and open popover menu with keyboard navigation

#### Possible solution
Add keyboard navigation directly to the `Breadcrumbs` component, or to `OverflowList`

**Comment by mm-wang** [comment]
I was able to click `Enter` on the ellipsis, so not sure if this is what this issue is referring to.
The items within the popover are disabled though, so that might be an issue with OverflowList.
